pydis_site/apps/api/migrations/0086_merge_20221231_0822.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.1.4 on 2022-12-31 08:22
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0085_add_thread_id_to_nominations'),
+        ('api', '0085_infraction_jump_url_text'),
+    ]
+
+    operations = [
```

Comment:
This file's the result of running `python manage.py makemigrations --merge`.

Reply:
Ah I see, I didn't know this existed.
Don't you think it'd be cleaner to have a separate one, with a different id ? instead of this merged format ?

Reply:
Similar merged migration files already exist in the project so imo it's fine.

Reply:
Sure, that's one way to view this.
Even though I don't fully agree, since having something that can or could've been improved doesn't mean we should do it again.
I think the conflict happened because of the migration I did on the nominations table, which has already been merged into main 2 days ago.
And the conflict happened just on the naming, and not on the model itself, and I still think it should be done on its own.

We can wait for others to see what they think about this.

Reply:
Migration files are not for humans anyway. However, I've gone ahead and generated a new migration file.